frontend/src/pages/components/gatheredList.js

```diff
@@ -51,35 +51,57 @@ export default function InfoGatheredList({pages}) {
     }
   }
 
+  //changing variables as per screensize
+  const [height, width] = useWindowSize();
+  const isSmall = width < 640;
+  const isMedium = width < 1024; //could also be 1007 instead of 1024 depending on standard used
+  const margin_left = isSmall? 1 : (isMedium ? 2 : 8);
+  const title_fontSize = isSmall? '8px' : (isMedium ? '12px' : '16px'); //use it later when making it suitable for medium and small sizes
+
+
   return (
     <div className={classes.root}>
-      <Box mt = {6} ml = {8}>
+      <Box mt = {6} ml = {margin_left}>
```

Comment:
I think you could achieve a similar thing by just using a percentage value, like `ml = "20%"`. If that doesn't have the effect you're looking for, then disregard.

Reply:
Okay, I will do that.

Reply:
Yes, it works, made the change.